drivers/FileHandle.h

```diff
@@ -25,62 +25,85 @@ namespace mbed {
 /** \addtogroup drivers */
 /** @{*/
 
-/** An OO equivalent of the internal FILEHANDLE variable
+
+/** Class FileHandle
+ *
+ *  An OO equivalent of the internal FILEHANDLE variable
  *  and associated _sys_* functions.
  *
- * FileHandle is an abstract class, needing at least sys_write and
+ *  FileHandle is an abstract class, needing at least sys_write and
```

Comment:
That's my view too - it's so generic and open-ended, it seems futile to try to pin it down through C++ classes.